server/controllers/pwd.controllers.js

```diff
@@ -0,0 +1,99 @@
+const asyncHandler = require('express-async-handler');
+const User = require('../models/user.modal');
+const nodemailer = require('nodemailer');
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcryptjs');
+
+module.exports.forgotPwdController = asyncHandler(async (req, res, next) => {
+  const { email } = req.body;
+  const user = await User.findOne({ email });
+
+  if (!user) {
+    res.status(400).send({ status: 'user not found' });
+    throw new Error('User not found');
+  }
+
+  const token = await user.createPWDresetToken();
+  const resetURL = `${req.protocol}://${req.get(
```

Comment:
//TODO: Here URL should be frontend, You can get from .env file name as FRONTEND_URL